widget/list_test.go

```diff
@@ -0,0 +1,161 @@
+package widget
+
+import (
+	"fmt"
+	"math"
+	"testing"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/driver/desktop"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/test"
+	"fyne.io/fyne/theme"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewList(t *testing.T) {
+	list := createList()
+	test.WidgetRenderer(list).(*listRenderer).Layout(fyne.NewSize(200, 1000))
+
+	template := fyne.NewContainerWithLayout(layout.NewHBoxLayout(), NewIcon(theme.DocumentIcon()), NewLabel("Template Object"))
+	firstItemIndex := test.WidgetRenderer(list).(*listRenderer).firstItemIndex
+	lastItemIndex := test.WidgetRenderer(list).(*listRenderer).lastItemIndex
+	visibleCount := len(test.WidgetRenderer(list).(*listRenderer).children)
+
+	assert.Equal(t, 1000, list.Length())
+	assert.GreaterOrEqual(t, list.MinSize().Width, template.MinSize().Width)
+	assert.Equal(t, list.MinSize(), test.WidgetRenderer(list).(*listRenderer).scroller.MinSize())
+	assert.Equal(t, 0, firstItemIndex)
+	assert.Equal(t, visibleCount, lastItemIndex-firstItemIndex+1)
+}
+
+func TestListResize(t *testing.T) {
```

Comment:
nit: `Test<struct-name>_<method-name>`
```suggestion
func TestList_Resize(t *testing.T) {
```

Reply:
These should be resolved now